bundles/plugins/org.bonitasoft.studio.designer/src/org/bonitasoft/studio/designer/ui/contribution/OpenUIDesignerCoolBarItem.java

```diff
@@ -60,6 +68,31 @@ protected OpenUIDesignerHandler getHandler() {
         return new OpenUIDesignerHandler();
     }
 
+    protected void openInformationDialogOnUiDesigner() {
+        final String displayDialog = getPreferenceStore().getString(HIDE_UIDESIGNER_INFO_DIALOG);
+        if (!"true".equals(displayDialog)) {
```

Comment:
I can, but doing it like avoid to have to initialize it and create a new Preference Initializer in this plugin